.github/workflows/golangci-lint.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.51.2
+          version: v1.53.3
```

Comment:
There's a parser issue with the `^time\.After$` regex that this fixes.

Reply:
Is it latest? No reason not to use latest, right?